src/passengerPickup/src/view/Components/defaultAgent.js

```diff
@@ -0,0 +1,117 @@
+export const agentPythonCodeFunction = `import random
+def getPlayersCommands(world):
+  foo = ['RIGHT', 'LEFT', 'UP','DOWN']
+  result = random.choice(foo)
+  print(result)
+	return result`;
+	
+export const agentJavascriptFunctionCode = `function getPlayersCommands(world) {
```

Comment:
why is the whole function in tick marks?

Reply:
This function need to be exported as a text